.github/workflows/e2e.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Set env
         run: |
           echo "VITE_SUI_BIN=$PWD/target/debug/sui" >> $GITHUB_ENV
-          echo "E2E_RUN_LOCAL_NET_CMD=(RUST_LOG=warn RUST_BACKTRACE=1 $(echo $PWD/target/debug/sui-test-validator) --with-indexer --pg-port 5432 --pg-db-name sui_indexer_v2 --graphql-host 127.0.0.1 --graphql-port 9125)" >> $GITHUB_ENV
+          echo "E2E_RUN_LOCAL_NET_CMD=(RUST_LOG=info RUST_BACKTRACE=1 $(echo $PWD/target/debug/sui) start --with-faucet --with-indexer --pg-port 5432 --pg-db-name sui_indexer_v2 --with-graphql --graphql-port 9125 --force-regenesis)" >> $GITHUB_ENV
```

Comment:
nit: how come the change in log level here?

Reply:
We shouldn't, some gremlin got into the keyboard. :D